client/client.py

```diff
@@ -0,0 +1,103 @@
+"""
+Base class which contains the basic methods to interaction with the write api of NodeBB.
+"""
+import json
+
+import requests
+import urlparse
+from django.conf import settings as django_settings
+
+
+class Client(object):
+    """
+    Client Class responsible to make connection with NodeBB and perform all calls.
+    """
+    def __init__(self):
+        self._configure()
+
+    def _configure(self):
+        self.endpoint = 'http://172.16.16.24:4568'
```

Comment:
i hope the `endpoint` will be acquired from settings in future.

Reply:
Done